File: Week2/day3/sum-all.py
# - Create a variable named `ai`
#   with the following content: `[3, 4, 5, 6, 7]`
# - Print the sum of the elements in `ai`

ai = [3, 4, 5, 6, 7]

def sum(list):
    num = 0
    for element in list:
        num += element
    return num

print(sum(ai))

File: Week7/Day3/sum-all.js
'use strict';
// - Create a variable named `ai` with the following content: `[3, 4, 5, 6, 7]`
// - Log the sum of the elements in `ai` to the console

var ai = [3, 4, 5, 6, 7];

function sumElements(list){
	var sumList = 0;
	for (var i = 0; i < list.length; i++){
		sumList += list[i];
	}
	return sumList;
}

console.log(sumElements(ai))